### DISCUSSION TITLE
MQTT problem

### DISCUSSION THREAD

**Discussion by radoslawo81** [AllskyTeam/allsky-modules#62] [Q&A]
Hello
Im trying solve the problem but unsuccessfully.
_GENERAL:_
AS_CLOUDAMBIENT,AS_GAIN,AS_SQM

_MQTT:_
http://192.168.1.33
1883
allsky
radoslawo
radoslawo

If I write in terminal:
mosquitto_sub -d -t allsky -u radoslawo -P radoslawo
nothing happen(Allsky not sending message)

but if I write manual message:
mosquitto_pub -d -t allsky -m "Hello wxcvorld!" -u radoslawo -P radoslawo
Im receiving the message.

Please help me if you could.

**Comment by Alex-developer** [comment]
Are you trying to use the module to publish data?

**Reply by radoslawo81** [reply]
Yes

**Reply by PiotrNomad** [reply]
I hound two problems in the MQTT modules:
  1. publish data requested SSL connection. See logs for error message.
  2. Paho has been updated and the see: https://github.com/AllskyTeam/allsky-modules/issues/120 

I manage to run it in my setup, but it require some changes to the code.
I hope this can help. 🤔

**Comment by Alex-developer** [comment]
Have you tested the latest version of the module? I have just installed it and the lastest version of mosquitto. 

I am not using SSL locally so I have that option unchecked but I am getting published data both with and without username/password
